sunpy/net/scraper.py

```diff
@@ -174,73 +173,7 @@ def _URL_followsPattern(self, url):
         """
```

Comment:
Do we need this method if all it calls is parse and a url?

Reply:
i don't think we do, i wanted to just confirm since it does add slightly in terms of readability

Reply:
How so?

Reply:
i just realised my reply didnt make it somehow but i fixed that in a later commit!